Bottom: 3b558615d61c0132e4458b85bf113d9c07ee2c89
Top:    9dc26f9e75afe38a3a828b47fba818842deef383
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 13:56:37 -0500

Refresh of wiki-update-entrypoint

---
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
